device/accton/x86_64-accton_as7716_32x-r0/Accton-AS7716-32X/sai.profile

```diff
@@ -0,0 +1,2 @@
+SAI_INIT_CONFIG_FILE=/etc/bcm/th-as7716-32x100G.config.bcm
```

Comment:
Could you follow the [Porting Guide](https://github.com/Azure/SONiC/wiki/Porting-Guide) and remove the config.bcm from the pull request and send it directly to me after verifying the file meets the requiremetns?